lib/evm/src/lib.rs

```diff
@@ -1,13 +1,13 @@
 mod client;
 #[allow(clippy::all)]
+#[rustfmt::skip]
```

Comment:
directs `clippy` and `rustfmt` to ignore `contract.rs` without relying on nightly features